packages/react/src/tabs/panel/TabsPanel.tsx

```diff
@@ -47,10 +47,14 @@ export const TabsPanel = React.forwardRef(function TabPanel(
     render: render ?? 'div',
     className,
     state,
-    extraProps: { ...other, children: hidden && !keepMounted ? undefined : children },
+    extraProps: { ...other, children },
```

Comment:
```suggestion
    extraProps: other,
```
This should be enough, no need to destructure children anymore ~